src/pages/groupComparison/ClinicalData.tsx

```diff
@@ -254,31 +255,42 @@ export default class ClinicalData extends React.Component<IClinicalDataProps, {}
         }
     });
 
-    private readonly groupSampleDataPromise = remoteData({
-        await: () => [this.props.store.sampleKeyToActiveGroups, this.props.store.uidToGroup, this.props.store._originalGroups],
+    public readonly groupSampleAxisData = remoteData({
```

Comment:
I still think `groupSample` is confusing. even just `groupMembershipAxisData` would be more clear.